docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -48,66 +48,91 @@ Let us look at this feature in the context of a use case. Say that a large but i
-
-
-```
-CREATE TABLE transactions_eu 
-    PARTITION OF transactions 
-      (user_id, account_id, geo_partition, account_type, 
-       amount, txn_type, created_at,
-       PRIMARY KEY (user_id HASH, account_id, geo_partition))
-    FOR VALUES IN ('EU');
-
-CREATE TABLE transactions_india 
-    PARTITION OF transactions
-      (user_id, account_id, geo_partition, account_type, 
-       amount, txn_type, created_at,
-       PRIMARY KEY (user_id HASH, account_id, geo_partition))
-    FOR VALUES IN ('India');
-
-CREATE TABLE transactions_default 
-    PARTITION OF transactions
-      (user_id, account_id, geo_partition, account_type, 
-       amount, txn_type, created_at,
-       PRIMARY KEY (user_id HASH, account_id, geo_partition))
-    DEFAULT;
+1. Create the parent table.
+
+    ```sql
+    CREATE TABLE transactions (
+        user_id   INTEGER NOT NULL,
+        account_id INTEGER NOT NULL,
+        geo_partition VARCHAR,
+        account_type VARCHAR NOT NULL,
+        amount NUMERIC NOT NULL,
+        txn_type VARCHAR NOT NULL, 
+        created_at TIMESTAMP DEFAULT NOW()
+    ) PARTITION BY LIST (geo_partition);
+    ```
+
+1. Create tablespaces for each region.
+
+    ```sql
+    CREATE TABLESPACE eu_central_1_tablespace WITH (
+      replica_placement='{"num_replicas": 3, "placement_blocks":
+      [{"cloud":"aws","region":"eu-central-1","zone":"eu-central-1a","min_num_replicas":1},
+      {"cloud":"aws","region":"eu-central-1","zone":"eu-central-1b","min_num_replicas":1},
+      {"cloud":"aws","region":"eu-central-1","zone":"eu-central-1c","min_num_replicas":1}]}'
+    );
+
+    CREATE TABLESPACE us_west_2_tablespace WITH (
+      replica_placement='{"num_replicas": 3, "placement_blocks":
+      [{"cloud":"aws","region":"us-west-2","zone":"us-west-2a","min_num_replicas":1},
+      {"cloud":"aws","region":"us-west-2","zone":"us-west-2b","min_num_replicas":1},
+      {"cloud":"aws","region":"us-west-2","zone":"us-west-2c","min_num_replicas":1}]}'
+    );
+
+    CREATE TABLESPACE ap_south_1_tablespace WITH (
+      replica_placement='{"num_replicas": 3, "placement_blocks":
+      [{"cloud":"aws","region":"ap-south-1","zone":"ap-south-1a","min_num_replicas":1},
+      {"cloud":"aws","region":"ap-south-1","zone":"ap-south-1b","min_num_replicas":1},
+      {"cloud":"aws","region":"ap-south-1","zone":"ap-south-1c","min_num_replicas":1}]}'
+    );
+    ```
+
+1. Next, create one partition per desired geography under the parent table. Here, you create three table partitions: one for the EU region called `transactions_eu`, another for the India region called `transactions_india,` and a third default partition for the rest of the regions called `transactions_default`.
```

Comment:
Also we need to mention that we are creating them in their appropriate geographic locations using tablespaces

Reply:
Done